packages/client/tiptap/extensions/database/TextCell.tsx

```diff
@@ -0,0 +1,18 @@
+import * as Y from 'yjs'
+import {Input} from '../../../ui/Input/Input'
+import {useYText} from './hooks'
+
+export const TextCell = ({text}: {text: Y.Text}) => {
+  const value = useYText(text)
+  return (
+    <Input
+      value={value}
+      className='w-full border-none'
+      onChange={(e) => {
+        // TODO this is wrong
```

Comment:
-1 I'm going to make the argument that we should use primitives instead of YText for cell values. YText is basically a YArray, where you can add/remove individual chars. We just want to replace the whole thing! This'll simplify the transactions, eliminate the need for cell diffing, etc.